src/features/PersonList/index.js

```diff
@@ -52,11 +42,12 @@ const PersonList = () => {
           <Pagination
             currentPage={currentPage}
             minPageLimit={1}
-            maxPageLimit={100}
+            maxPageLimit={400}
+            url={toPersonList()}
           />
         </Container>
       ) : (
-        <ErrorPage />
```

Comment:
as and this

Reply:
It's the same thing I comment on for Main Page